src/access/examples/ExampleAccessControlDelegate.cpp

```diff
@@ -483,6 +546,100 @@ class EntryStorage
         index = found ? toIndex : ArraySize(acl);
     }
 
+public:
+    static const chip::TLV::Tag kTagInUse       = chip::TLV::ContextTag(1);
+    static const chip::TLV::Tag kTagFabricIndex = chip::TLV::ContextTag(2);
+    static const chip::TLV::Tag kTagAuthMode    = chip::TLV::ContextTag(3);
+    static const chip::TLV::Tag kTagPrivilege   = chip::TLV::ContextTag(4);
+    static const chip::TLV::Tag kTagSubjects    = chip::TLV::ContextTag(5);
+    static const chip::TLV::Tag kTagTargets     = chip::TLV::ContextTag(6);
+    static const size_t kStorageBufferSize      = 192;
+
+    CHIP_ERROR Serialize(chip::PersistentStorageDelegate * storage, const char * key)
+    {
+        uint8_t buffer[kStorageBufferSize] = { 0 };
+        chip::TLV::TLVWriter writer;
+        writer.Init(buffer, sizeof(buffer));
```

Comment:
```suggestion
        writer.Init(buffer);
```
Size gets inferred via template magic.  ;)